src/api/environment.cc

```diff
@@ -258,6 +261,26 @@ void FreePlatform(MultiIsolatePlatform* platform) {
   delete platform;
 }
 
+MaybeLocal<Object> GetPerContextExports(Local<Context> context) {
```

Comment:
If this is not exposed in `node.h`, this can be implemented in somewhere like `env.cc`.

Reply:
I don’t think `env.cc` would be the right place for that – it’s tied to `NewContext()` and not any of our `Environment`/`IsolateData` classes, right? But yeah, this is not the ideal location for it…